src/pylhe/__init__.py

```diff
@@ -254,8 +254,13 @@ def read_lhe_init(filepath):
                         try:
                             wg_type = child.attrib["type"]
                         except KeyError:
-                            print("weightgroup must have attribute 'type'")
-                            raise
+                            try:
+                                wg_type = child.attrib["name"]
+                            except KeyError:
+                                print(
+                                    "weightgroup must have attribute 'type' or 'name'"
+                                )
+                                raise
```

Comment:
Is it possible to cover these in the tests easily? Or would that require a lot of extra work?

Reply:
I think some bits will require special files or a fair workaround. Some may require a proper follow-up but I think I can improve this MR a bit more in any case. Give me until tomorrow and I will summarise the situation - takes longer because (1) I'm no expert in LHE files and (2) this work is something I cannot prioritise.

Reply:
@eduardo-rodrigues That's fine. Let's just push this back to `v0.5.0` then. I'll make an Issue from this discussion.